src/applications/personalization/dashboard/components/debts/GenericDebtCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import recordEvent from '~/platform/monitoring/record-event';
+import CTALink from '../CTALink';
+
+const GenericDebtCard = () => {
+  const cardHeader = 'Overpayments';
+
+  const linkText = 'Manage your VA debt';
+
+  // Linking to existing applications
+  const linkDestination = '/manage-va-debt/summary/debt-balances';
+
+  return (
+    <va-card
+      show-shadow
+      className="vads-u-display--flex vads-u-width--full vads-u-flex-direction--column vads-u-justify-content--space-between vads-u-align-items--flex-start vads-u-padding--1"
+      data-testid="debt-card"
+    >
```

Comment:
Can we match the style of the existing cards (`./DebtsCard.jsx`) as well? 

They look a little wonky side by side
![image](https://github.com/user-attachments/assets/6f813955-3312-40c5-a567-029351d91d71)

Reply:
Hmm its already using the same class, will mess with it.